index.d.ts

```diff
@@ -0,0 +1,43 @@
+export interface Options {
+	/**
+	 * Return other users' processes as well as your own.
```

Comment:
It's not a method, so it doesn't return anything. It *is* the value.
```suggestion
	 * Other users' processes as well as your own.
```